aiida_raspa/workchains/aiida_base.py

```diff
@@ -0,0 +1,437 @@
+"""
```

Comment:
So the idea behind the usage of this [`aiida_base` workchain](https://github.com/aiidateam/aiida-cp2k/blob/develop/aiida_cp2k/workchains/aiida_base_restart.py)  that we keep it unchanged. The usage of it allowed to make such a small [`Cp2kBaseWorkChain`](https://github.com/aiidateam/aiida-cp2k/blob/develop/aiida_cp2k/workchains/base.py). So my suggestion would be to copy `aiida_base` entirely and then to create a new file with much smaller `RaspaBaseWorkChain` in it.